frontend/app/routes/protected/person-case/review.tsx

```diff
@@ -39,6 +51,7 @@ export async function loader({ context, request }: Route.LoaderArgs) {
 function validateInPersonSINCaseSession(
   session: AppSession,
   request: Request,
+  locale: Language = 'en',
```

Comment:
I don't think this should have a default value. There are no times when this function should ever be called without a valid language.

also:

```suggestion
  language: Language,
```

Reason: `'en'` and `'fr'` are not locales.